### DISCUSSION TITLE
Latest QDMR is mis-detecting my Anytone 878-uvii

### DISCUSSION THREAD

**Discussion by andrewschott** [hmatuschek/qdmr#483] [General]
When I fire up qdmr, it sees my cabling and such on the correct port (ttyUSB0), but sees the AT-878uvii as a Baofeng DR-1801UV now.  If I try to just use that, it still will not read from the radio.

Any tips?
TIA,
Andrew.  KD9TEA

**Comment by hmatuschek** [comment]
No, it is not. If no compatible device is detected (for what reason at all) but a serial port is found, QDMR asks, if one of the devices it cannot detect, is connected to that port. In this case there is only one supported device, that cannot be detected reliably, the DR-1801UV.
